src/runtime_src/core/tools/xbmgmt2/SubCmdProgram.cpp

```diff
@@ -730,6 +733,36 @@ SubCmdProgram::execute(const SubCmdOptions& _options) const
     throw xrt_core::error("uuid does not match BLP");
   }
 
+  // -- process "user" option ---------------------------------------
+  if(!xclbin.empty()) {
+    XBU::verbose(boost::str(boost::format("  xclbin: %s") % xclbin));
+    //only 1 card and name
+    if(deviceCollection.size() > 1)
+      throw xrt_core::error("Please specify a single device");
+    auto dev = deviceCollection.front();
+
+    std::ifstream stream(xclbin, std::ios::binary);
+    if (!stream)
+      throw xrt_core::error(boost::str(boost::format("Could not open %s for reding") % xclbin));
```

Comment:
What does the color "red" have to do with reading in the file....LOL

Reply:
Hehe :P